compiler/qsc_eval/src/state/tests.rs

```diff
@@ -566,14 +566,14 @@ fn assert_latex_for_decimal(expected: &Expect, number: f64, render_one: bool) {
 fn check_get_latex_for_decimal() {
     assert_latex_for_decimal(
         &expect!([r#"
-            "0.25"
+            "0.2500"
         "#]),
         0.25,
```

Comment:
@tcNickolas , I am talking about these test cases. I tried out some basic superpositions, they are rendering fractions. 

Unrelated bug, for large states, it seems we omit the part of $|\psi\rangle = ... $.  

```qsharp
namespace Sample {

    open Microsoft.Quantum.Diagnostics;
    @EntryPoint()
    operation Superposition() : Unit {
        // Qubits are only accesible for the duration of the scope where they
        // are allocated and are automatically released at the end of the scope.
        
        let N=8;
        use qubit = Qubit[N];

        for i in 0..N-1{
            H(qubit[i]);
        }

        DumpMachine();

        ResetAll(qubit);
    }
}
```

Reply:
Indeed, you were right. I kind of digged into code to find this entry point :-). Thanks @tcNickolas for clearing my doubt.
https://github.com/microsoft/qsharp/blob/28817e401664ad2ab4f53f526f37ac00e310dd69/compiler/qsc_eval/src/state.rs#L344-L348